src/services/rsvp/rsvp-router.test.ts

```diff
@@ -0,0 +1,114 @@
+import { describe, expect, it, beforeEach } from "@jest/globals";
+import { TESTER, getAsAdmin, getAsAttendee, getAsStaff, putAsApplicant } from "../../testTools.js";
+import { DecisionStatus, DecisionResponse } from "../../database/decision-db.js";
+import Models from "../../database/models.js";
+
+// Before each test it'll add this to the database
+beforeEach(async () => {
+    Models.initialize();
+    await Models.DecisionInfo.create({
```

Comment:
Convert this object into a global variable! You're using it elsewhere anyways, and it will make this much cleaner to test. See my other comment below for how to convert .toHavePropertys into .toMatchObject